src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/MergedCalendar.java

```diff
@@ -15,19 +15,19 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
-    private String userName;
+    private String username;
```

Comment:
@RedRiot instead of a username there should be an association to the `UserModel user` and the `UserModel` should have a `List<MergedCalendar> mcs`